docs/ManualDeploymentGuideScalarDBServerOnAKS.md

```diff
@@ -1,72 +1,55 @@
-# Deploy Scalar DB Server on AKS (Azure Kubernetes Service)
+# Deploy ScalarDB Server on AKS (Azure Kubernetes Service)
 
-Scalar DB Server is a gRPC server that implements the Scalar DB interface. With Scalar DB Server, you can use Scalar DB features from multiple programming languages that are supported by gRPC.  
-
-We can deploy Scalar DB Server on any Kubernetes services. This document explains how to deploy Scalar DB Server on AKS.  
+ScalarDB Server is a gRPC server that implements the ScalarDB interface. With ScalarDB Server, you can use ScalarDB features from multiple programming languages that are supported by gRPC. We can deploy ScalarDB Server on any Kubernetes services. This document explains how to deploy ScalarDB Server on AKS.
 
 ## What we create
 
-In this guide, we create the following environment on your Azure account.  
-
-![image](images/scalardbserver-aks-diagram.png)
-
-// TODO: Update the figure based on the latest document
-
-## Step 1. Subscribe to Scalar DB Server in Azure Marketplace
-
-You can get the Scalar DB Server container image from [Azure Marketplace](https://azuremarketplace.microsoft.com/en/marketplace/apps/scalarinc.scalardb). First, you need to subscribe to it. Please refer to the following document to subscribe to Scalar DB Server in Azure Marketplace.  
+In this guide, we create one of the following two environments on your Azure account. The difference is application deployment.
```

Comment:
ScalarDB Server has two options for its deployment.

1. Deploy user application on the same k8s cluster as ScalarDB Server deployment.
2. Deploy user application on other environments than the k8s cluster that users deploy ScalarDB Server.